src/Fixture/Factory/ShopBillingDataExampleFactory.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\InvoicingPlugin\Fixture\Factory;
+
+use Sylius\Bundle\CoreBundle\Fixture\Factory\AbstractExampleFactory;
+use Sylius\Bundle\CoreBundle\Fixture\Factory\ExampleFactoryInterface;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Channel\Context\ChannelNotFoundException;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\InvoicingPlugin\Entity\ShopBillingData;
+use Sylius\InvoicingPlugin\Entity\ShopBillingDataAwareInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class ShopBillingDataExampleFactory extends AbstractExampleFactory implements ExampleFactoryInterface
+{
+    /**
+     * @var ChannelRepositoryInterface
+     */
+    private $channelRepository;
+
+    /**
+     * @var OptionsResolver
+     */
```

Comment:
The convention in this plugin is to use short docblocks, so `/** @var OptionsResolver */` (same above)